treeherder/etl/tasks/pulse_tasks.py

```diff
@@ -16,7 +17,10 @@ def store_pulse_jobs(job_list, exchange, routing_key):
     newrelic.agent.add_custom_parameter("exchange", exchange)
     newrelic.agent.add_custom_parameter("routing_key", routing_key)
 
-    JobLoader().process_job_list(job_list)
+    try:
+        JobLoader().process_job_list(job_list)
```

Comment:
I know this is pre-existing, but is job_list always only one job? If so perhaps we should s/job_list/job/ plus s/store_pulse_jobs/store_pulse_job/ ?

If not, then with this PR how is the case of "two jobs, but one has a revision which is unknown" case work? We'll retry both right?